bundle/manifests/grafana-operator.clusterserviceversion.yaml

```diff
@@ -304,17 +304,35 @@ spec:
         serviceAccountName: grafana-operator-controller-manager
       deployments:
       - label:
+          app.kubernetes.io/component: controller-manager
+          app.kubernetes.io/instance: grafana-operator-controller-manager
+          app.kubernetes.io/managed-by: olm
+          app.kubernetes.io/name: grafana-operator-controller-manager
+          app.kubernetes.io/part-of: grafana-operator
+          app.kubernetes.io/version: v5.6.0
           control-plane: controller-manager
         name: grafana-operator-controller-manager
         spec:
           replicas: 1
           selector:
             matchLabels:
+              app.kubernetes.io/component: controller-manager
+              app.kubernetes.io/instance: grafana-operator-controller-manager
+              app.kubernetes.io/managed-by: olm
+              app.kubernetes.io/name: grafana-operator-controller-manager
+              app.kubernetes.io/part-of: grafana-operator
+              app.kubernetes.io/version: v5.6.0
               control-plane: controller-manager
           strategy: {}
           template:
             metadata:
               labels:
+                app.kubernetes.io/component: controller-manager
+                app.kubernetes.io/instance: grafana-operator-controller-manager
+                app.kubernetes.io/managed-by: olm
+                app.kubernetes.io/name: grafana-operator-controller-manager
+                app.kubernetes.io/part-of: grafana-operator
+                app.kubernetes.io/version: v5.6.0
```

Comment:
Would prefer not to have version as a part of this. Then we will have to add logic to update the version.
I also don't think we gain anything by adding it.